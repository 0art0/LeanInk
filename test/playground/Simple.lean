set_option trace.Elab.info true

def add (x y : Nat) := x + y