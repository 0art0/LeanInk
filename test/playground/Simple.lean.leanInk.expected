[{"contents": "set_option trace.Elab.info true\n\ndef add (x y : Nat) := x + y",
  "_type": "text"}]